docs/tasks/includes/_rust-setup.md

```diff
@@ -0,0 +1,18 @@
+This is how to setup your code to use the Rust library.
+
+```rust
+use std::{
+    io::{Cursor, Write},
+    process::{Command, Stdio},
+};
+
+use anyhow::Result;
+use c2pa::{settings::load_settings_from_str, Builder, CallbackSigner, Reader};
+
```

Comment:
nit: Remove this blank line? I like to group imports by std/external/crate

Reply:
Removed.